hybrasyl/NumberCruncher.cs

```diff
@@ -0,0 +1,140 @@
+﻿using Hybrasyl.Castables;
+using Hybrasyl.Objects;
+using Hybrasyl.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using log4net;
+using Hybrasyl.Statuses;
+
+namespace Hybrasyl
+{
+
+    /// <summary>
+    /// This class is used to do a variety of numerical calculations, in order to consolidate those into
+    /// one place. Specifically, healing and damage, are handled here.
+    /// </summary>
+    /// 
+
+    static class NumberCruncher
+    {
+
+        public static ILog Logger = LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
+
+        // This is dumb, but it's a consequence of how xsd2code works
+        static private double _evalSimple(dynamic simple)
+        {
```

Comment:
private static for proper syntax.

Reply:
Fixed

Reply:
note: this occurs in many places, just referenced a single for simplicity